nservicebus/hosting/azure-functions/service-bus_assembly-scanner_ASBFunctions_[2.0,).partial.md

```diff
@@ -0,0 +1,3 @@
+### Assembly scanning
+
+The assembly scanner loads assemblies in the `bin` directory of the application, as well as the assemblies that are available at the function's runtime directory. The behavior can be customized by leveragin the [assembly scanner](/nservicebus/hosting/assembly-scanning.md) API. 
```

Comment:
```suggestion
The assembly scanner loads assemblies in the `bin` directory of the application, as well as the assemblies that are available at the function's runtime directory. The behavior can be customized by leveraging the [assembly scanner](/nservicebus/hosting/assembly-scanning.md) API. 
```

Reply:
> as well as the assemblies that are available at the function's runtime directory

Are these a part of the Azure Functions runtime? It might be worth clarifying what the difference is here. It may even be worth calling out why you might want to control the assembly scanner. Something like:

> The assembly scanner loads assemblies from two different locations:
> - The `bin` directory of the applictation containing function definitions and any dependencies.
> - The Azure Function runtime directory.
>
> If the assembly scanner attempts to load the same assembly  twice (usually because it is present in both locations) then an exception will be thrown.
> (ADD EXAMPLE OF EXCEPTION)
> The [assembly scanner](/nservicebus/hosting/assembly-scanning.md) API can be used to control which assemblies get scanned.

Reply:
@mikeminutillo  I reworded this section, see if it reads better now? feel free to suggest further changes.